### ISSUE TITLE
Page Templates e2e test failure

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#9484]
The page templates e2e test is[ producing an error ](https://github.com/google/web-stories-wp/pull/9289/checks?check_run_id=3954199933)that seems related to Percy. The test should be fixed and enabled. ![Screen Shot 2021-10-20 at 9.52.46 AM.png](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/a140a43b-be37-47fd-96e7-644e426ce527)

**Comment by swissspidy** [comment]
The error says there is a 404 response happening when dealing with page templates in the editor.

Check your console in the editor and you should spot this as well when performing the same actions.

Not related to Percy.

This seems to happen only with the page templates PR, so should get fixed there.

**Comment by BrittanyIRL** [comment]
So what's happening, we think, is that there was a missing screenshot for the bread template which up to now hasn't been an issue anywhere in the editor/anywhere where the 404 would be noticed, but since this PR is introducing screenshots for the default page templates it's now getting acknowledged. I've added the missing screenshot and moving forward this will error the same if further images are missing which is great! 

I just didn't think that an image not rendering would cause the e2e test to error out since it doesn't impact usability, so today I learned.  Once the above is verified i'll close this ticket. The test is re-enabled on the original PR.
